core-rust/state-manager/src/protocol/protocol_updates/definitions/anemone_definition.rs

```diff
@@ -36,37 +28,7 @@ impl ProtocolUpdateDefinition for AnemoneProtocolUpdateDefinition {
         Box::new(BatchedUpdater::new(
             new_protocol_version.clone(),
             Self::state_computer_config(network_definition),
-            AnemoneBatchGenerator,
+            ScryptoEntriesBatchGenerator::new(&ANEMONE_ENTRIES),
```

Comment:
I appreciate the refactoring - but I'm not so keen on this `ProtocolUpdateEntry` formalism in the engine, and was going to refactor it next week to be more flexible / aligned with the batching flexibility we used to have in the node, and that we might need in a pinch in certain expedited protocol update scenarios.

Basically what I'm saying is I'm happy to keep this refactor for now, but it'll maybe need to be reverted in a week or two :D

Reply:
I thought that `ProtocolUpdateEntry` (or rather, its surrounding infra) nicely groups the PU parts (that previously used to be "a bunch of `fn`s in neighbouring files". But if you want to refactor further, it is fine - as long as the "PU parts" (whatever representation you end up with) are easy to reference from Node.

(one part of that refactor _[for your consideration]_ could be to "actually name the transactions in Engine repo", which would simplify the Node's declarative part that you noticed at https://github.com/radixdlt/babylon-node/pull/907#discussion_r1568074902)